packages/next/src/client/components/react-dev-overlay/server/middleware-turbopack.ts

```diff
@@ -51,18 +69,19 @@ export async function batchedTraceSource(
         lineNumber: frame.line ?? 0,
         column: frame.column ?? 0,
         methodName: frame.methodName ?? '<unknown>',
-        ignored: shouldIgnorePath(frame.file),
+        ignored: shouldIgnorePath(file),
         arguments: [],
       },
       source: null,
     }
   }
 
   let source = null
-  const originalFile = sourceFrame.originalFile
+  const originalFile = sourceFrame.originalFile ?? sourceFrame.file
```

Comment:
It's no longer the original location which means we also try to show source frames for unsourcemapped frames by entering `if (originalFile && !ignored) {` on L87 below. Is this intended?

At the very least, we should rename this variable now to make that clear.